src/utils/sformat.fs

```diff
@@ -1194,7 +1194,7 @@ namespace Microsoft.FSharp.Text.StructuredFormat
             | :? bool   as b -> (if b then "true" else "false")
             | :? char   as c -> "\'" + formatChar true c + "\'"
             | _ -> try  let text = obj.ToString()
-                        text
+                        if text = null then "" else text
```

Comment:
isNull is not available in proto, and pattern matching here seems ugly. :) 
Surely we don't have to do everything FSLint says? FSLint would probably find tons of issues in this code base.